spm-monitor/src/main/java/com/sematext/spm/client/command/UpdateInstrumentationSettingsHandler.java

```diff
@@ -44,16 +41,6 @@ public class UpdateInstrumentationSettingsHandler implements CommandHandler {
   @Override
   public Cancellable handle(TCommand command, ResponseCallback callback) {
     final TUpdateInstrumentationSettings thriftSettings = new TUpdateInstrumentationSettings();
-    try {
-      ThriftUtils.binaryProtocolDeserializer().deserialize(thriftSettings, command.getRequest());
```

Comment:
Are we sure we want to completely delete deserialisation of the command?

Reply:
My thought was that we wouldn't need the command anyway, but you're right, this would make the call to this method fail. Reverting this fragment